### PR TITLE
Update Golden Layout to V2.5.0

### LINKED ISSUE #120: Update goldenlayout
they completely rewrote it and the "old" way of setting it up no longer works with any version after the one we're using.

### PR DESCRIPTION
**Pull request by caleb-mabry**
Updates Golden Layout to use v2.5.0.
Resolves #120 
Had to update some of our custom styling since there were element differences in the header.
image

### CODE REVIEW

**Comment by stonedDiscord** [comment]
i get a charselect on a white background
and a CSP error about the media-src

File: public/client.html [hunk 1]
```diff
@@ -20,11 +20,7 @@
     <meta property="twitter:url" content="http://web.aceattorneyonline.com/">
     <meta property="twitter:image" content="https://repository-images.githubusercontent.com/78860508/89fcba80-aafd-11e9-80db-c5b10c01aba9">
 
-	<meta http-equiv="Content-Security-Policy" content="script-src-elem 'self' 'unsafe-inline' https://ajax.googleapis.com;
-														script-src 'self' 'unsafe-inline' 'unsafe-eval' https://ajax.googleapis.com;
-														style-src-elem 'self' 'unsafe-inline' https://golden-layout.com;
-														style-src 'self' 'unsafe-inline' https://golden-layout.com;
```
Comment (oldmud0) [inline public/client.html:26]: csp is meant to block xss attacks so we should be diligent about keeping our `script-src` rules accurate. if we don't add any, there will be no protection from XSS

Comment (caleb-mabry) [reply public/client.html]: I've added them back.

**Comment by stonedDiscord** [comment]
i forgot to run npm install

looks good

**Pull request merged**